first-aid.lic

```diff
@@ -10,39 +10,36 @@ class FirstAid
   def initialize
     settings = get_settings
     @chart_data = get_data('anatomy-charts').first_aid_charts
-    @textbook = settings.textbook
-    if @textbook
+    if settings.textbook
       @booktype = 'textbook'
       textbook_charts
     else
       @booktype = 'compendium'
       compendium_charts
     end
   end
-  
   def compendium_charts
     bput('get my compendium', 'You get')
     bput('look my compendium', '^The compendium lies open to the section on .* physiology')
     pause
     charts = reget(40).grep(/^   .+$/).map(&:strip)
-
-    charts_to_read = charts.select { |chart| @chart_data[chart]['scholarship'] <= DRSkill.getrank('Scholarship') }
+    charts_to_read = @chart_data.select { |chart| @chart_data[chart]['scholarship'] <= DRSkill.getrank('Scholarship') }
```

Comment:
`chart` and `@chart_data[chart]` should be the same thing. You can simplify this by just using `chart`. Also line 27 and 28.